src/misc/string_utils.cpp

```diff
@@ -180,12 +180,61 @@ std::vector<std::string> split(const std::string &seq)
 	return words;
 }
 
+std::string join_with_commas(const std::vector<std::string>& items)
+{
+	// skip the empties: we don't want to see them
+	auto is_not_empty = [](const std::string& s) { return !s.empty(); };
+	const auto num_items = std::count_if(items.begin(), items.end(), is_not_empty);
```

Comment:
Kinda okay in this case, but you can compose functions better if one function does one thing. So stripping could be extracted into a `strip_empty_strings` function or something, then you can compose them. Having said that, if you use this function a lot, that becomes tiresome...

Reply:
Indeed; hmm.. I included this bit only defensively knowing that /maybe/ someone might pass it vectors with empty strings (and they would get output like `", , and ."` 

On second thought, I think it's even cleaner that we expect inbound vectors to have no empties, so I will assert that in the loop. (And the filenames being added by imgmount and certainly non-empty because they've passed various checks before hand).